controllers/storagecluster/provider_server.go

```diff
@@ -27,11 +27,11 @@ import (
 )
 
 const (
-	ocsProviderServerName               = "ocs-provider-server"
-	providerAPIServerImage              = "PROVIDER_API_SERVER_IMAGE"
-	onboardingSecretGeneratorImage      = "ONBOARDING_SECRET_GENERATOR_IMAGE"
-	onboardingJobName                   = "onboarding-secret-generator"
-	onboardingTicketPublicKeySecretName = "onboarding-ticket-key"
+	ocsProviderServerName                   = "ocs-provider-server"
+	providerAPIServerImage                  = "PROVIDER_API_SERVER_IMAGE"
+	rotateOnboardingKeypairsImage           = "ROTATE_ONBOARDING_KEYPAIRS_IMAGE"
```

Comment:
1. "Key pairs: are two different words as such you should camelcase both. 
2. The terminology here is work. These are not onboarding keys, these are onboarding validation keys.
3. We should not name a component with an action (e.g. "ROTATE") we should name as as a subject.
4. The job technically does not rotate the keys, it regenerates the keys.

With that in mind I suggest calling the component `onboardingValidationKeysGenerator` and all other names should derive from it. 

For example:

```suggestion
	onboardingValidationKeysGeneratorImage           = "ONBOARDING_VALIDATION_KEYS_GENERATOR_IMAGE"
```